packages/gestalt/src/FormActions.js

```diff
@@ -0,0 +1,17 @@
+// @flow strict
+import * as React from 'react';
+import Box from './Box.js';
+import Row from './Row.js';
+import Button from './Button.js';
+
+export default function FormActions(): React.Node {
+  return (
+    <Box padding={2}>
+      <Row>
+        <Button text="Cancel" />
```

Comment:
@chrislloyd Seems like this was a test to add a layout component - we should work with design to get these added. We should also not add literal strings.

Going to close this PR for now but feel free to re-open.